scripts/test-export-ciphers-rejected.py

```diff
@@ -118,9 +123,11 @@ def main():
                        CipherSuite.TLS_RSA_WITH_AES_128_CBC_SHA,
                        CipherSuite.TLS_EMPTY_RENEGOTIATION_INFO_SCSV]
             node = node.add_child(ClientHelloGenerator(ciphers, version=prot))
-            if prot == (3, 0) and not ssl3:
-                node = node.add_child(ExpectAlert(AlertLevel.fatal,
-                                                  AlertDescription.protocol_version))
+            if prot < min_ver:
```

Comment:
This test isn't covered by the description of the test:
```
Test if export grade ciphers are rejected by server
```
It looks like a test for rejecting unsupported protocol versions with the right alert.

Reply:
because it is checking if the server can't be forced to use export grade if the protocol version advertised by client is lower than the lowest version supported by server

but for such Client Hello, the server can reply with either of two alerts – there is no general precedence order for alert descriptions